### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by Sophichi** [Norbyte/ositools#114]
Is there any way to turn off auto-updates for the script extender?

**Comment by PinewoodPip** [comment]
Yes, you can lock it to a specific version by editing the `ScriptExtenderUpdaterConfig.json` file in the `bin` folder (not present by default, you might have to create it).

Example:
```json
{
    "UpdateChannel": "Devel",
    "TargetVersion": "58.0.0.0",
    "TargetResourceDigest": "2e0bed670d2e172e14af13b4a6265dd341ef2fbe276f6bee8f8392de95e9fc8f",
    "Debug": true
}
```

This is supported in v5+ of the updater.

The digests can be found at https://dbn4nit5dt5fw.cloudfront.net/Channels/Devel/Manifest2.json (replace Devel with Release for versions from the stable branch) and timestamps are in Windows FILETIME (https://learn.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-filetime)

**Comment by MrSmite** [comment]
This doesn't seem to work with version 59. I created the following and already have the files from the last extender update, but it still suspends client thread while checking for updates:

```json
{
    "UpdateChannel": "Release",
    "TargetVersion": "59.0.0.0",
    "TargetResourceDigest": "f92bc1e96ef0a968cc96b9e36a31e5b2b399295c5164e343fa4846891bbae758",
    "Debug": false
}
```
